framework/src/executioners/Transient.C

```diff
@@ -126,6 +128,15 @@ validParams<Transient>()
       std::numeric_limits<unsigned int>::max(),
       "Maximum number of times to update XFEM crack topology in a step due to evolving cracks");
 
+  params.addParam<Real>("relaxation_factor",
+                        1.0,
+                        "Fraction of newly computed value to keep."
+                        "Set between 0 and 2.");
+  params.addParam<std::vector<std::string>>("relaxed_variables",
+                                            std::vector<std::string>(),
+                                            "List of variables to relax during Picard Iteration");
+
+  params.addParamNamesToGroup("relaxation_factor relaxed_variables", "Picard");
```

Comment:
The group `Picard` has existed. You may just insert these two new parameters into that list.